inlineplz/registry.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+# glob patterns for what language is represented by what type of files.
+PATTERNS = {}
+# linter configs. add new tools here.
+LINTERS = {}
+
+
+def register_pattern(pname, patterns):
+    if pname in PATTERNS:
+        print(u"WARNING: duplicate pattern: {}".format(pname))
+    PATTERNS[pname] = patterns
+
+
+def get_patterns():
```

Comment:
These are already accessible directly, no reason for getters unless we name-mangle the variables and return copies instead of the mutable versions.